pkg/server/healthz.go

```diff
@@ -16,6 +16,10 @@ import (
 	"k8s.io/klog/v2"
 )
 
+const (
+	readHeaderTimeout = 5 * time.Second
```

Comment:
should we move this to utils package so it's shared?

Reply:
If we use constants from `utils` package, it'll lead to import cycle, that's why I defined them as local constants.